src/server/game.js

```diff
@@ -158,6 +251,12 @@ let game = function() {
 				// If we had hit the marble limit on the previous round, that's no longer true
 				this.limitReached = false;
 
+				// Create new round data
+				_round = _generateNewRoundData();
```

Comment:
I think generating new round data would be better when the game state changes to `waiting` in `setCurrentGameState`.
Same with `_isWaitingForEntry`, and possibly other stuff throughout the code that happens when the state changes.